src/main/java/net/openhft/chronicle/bytes/BytesStore.java

```diff
@@ -397,12 +397,19 @@ default char charAt(int index)
     }
 
     /**
-     * Not supported.
+     * This method is inherited from CharSequence.
+     *
+     * @return  {@link BytesStore} of (end - start) length
+     * containing chars from #charAt(start) to #charAt(end - 1), inclusive.
+     * Memory storage type (heap or native) is preserved.
      */
     @NotNull
     @Override
     default CharSequence subSequence(int start, int end) {
```

Comment:
We could return a more specific CharSequence namely Bytes

Reply:
Agreed, changed return type to BytesStore (Bytes would be incorrect)